.github/workflows/windows-msvc.yml

```diff
@@ -160,54 +159,6 @@ jobs:
           cd tests
           sed -i '/AT_SETUP(\[runtime check: write to internal storage (1)\])/a AT_SKIP_IF(\[true\])' testsuite.src/run_misc.at
 
-      - name: Adjust testsuite for Debug target
```

Comment:
I tackled most of these issues, only two remain and are explicitly skipped in the workflow file (beware when rebasing)

Reply:
Do you mean I should introduce the changes to this file from https://github.com/ddeclerck/gnucobol/commit/8ae65ac260aef95e850972b72eef1bdef1ab0ed3?

Reply:
That's the idea. Though if you just rebase on `gcos4gnucobol-3.x` you'll get those changes.

Reply:
I still see all the old fixes on that branch. Maybe there's something missing…

https://github.com/OCamlPro/gnucobol/blob/gcos4gnucobol-3.x/.github/workflows/windows-msvc.yml

Reply:
Ah, yeah, forgot to update the MSVC workflow after the SVN merge. Should be fine now.